lessons/en/misc/nimble_publisher.md

```diff
@@ -1,5 +1,5 @@
 %{
-  version: "1.0.2",
+  version: "2.0.0",
```

Comment:
@scubass can provide some additional context on why we need this version change?

Reply:
Phoenix changed a lot and is not compatible with the other version code, so I thought that the version should reflect the major update to the library.

Reply:
This version reflects the _lesson's_ version to notify translators when changes need to be made.